Components/Sources/SwiftUI/View/Buttons/SecondaryButtonView.swift

```diff
@@ -8,6 +8,8 @@
 import SwiftUI
 
 public struct SecondaryButtonView: View {
+    @SwiftUI.Environment(\.project) private var project
```

Comment:
Not necessary here. (see ProjectSecondaryButtonStyle)

Reply:
42865f8bd05adb2caceecc96c298b8d9d7bc956d